channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt

```diff
@@ -91,6 +123,7 @@ class Session internal constructor(
         /**
          * Set the username and password used to connect to the server and channel
          */
+		@Suppress("unused") // It's used by SDK consumers
         fun setUsername(username: String?, password: String?): Builder {
```

Comment:
Strangely this module and some of the other old ones use spaces as tabs, but the new changes in the same files are actually properly tabbed. Would you run the auto formatting on all the files? Option+Command+L to format an individual file.

Reply:
Would appreciate if it got auto-formatted.

Reply:
Reformatting all files that this pull request touched.